http/data-url.json

```diff
@@ -12,15 +12,18 @@
           "chrome_android": "mirror",
           "edge": {
             "version_added": "12",
-            "notes": "The maximum size supported is 4GB"
+            "notes": "Before Edge 79, the maximum size supported is 4GB."
           },
           "firefox": {
             "version_added": true
           },
           "firefox_android": "mirror",
           "ie": {
             "version_added": "8",
-            "notes": "The maximum size supported is 32kB"
+            "notes": [
+              "Since Internet Explorer 9, the maximum size supported is 4GB.",
+              "In Internet Explorer 8, the maximum size supported is 32kB."
```

Comment:
Hmm, this makes me wonder if the point of these notes is really just to document the very limited size in IE 8?

The 4GB limit doesn't really seem worth documenting TBH. But if it is, we should test it to confirm, and probably some other browser also has such a limit without a note...

Reply:
I don't really think it's significant enough to document, but I'd like to leave the notes for now, since determining whether they're removed feels out-of-scope for this PR (and I'd like to unblock the linter PR as soon as possible).